### ISSUE TITLE
Is it possible not to use a file for configuration and do it at runtime?

### ISSUE THREAD

**Issue by matiasperrone** [zoho/zcrm-php-sdk#3]
I need to be able to load a dynamic configuration in a multi-server enviroment.
I'd like to use a database for setting the configuration.

**Comment by hilobok** [comment]
Yeah, having config file inside `vendor` directory not a good idea at all
